.github/workflows/api-integration-tests.yml

```diff
@@ -104,6 +104,19 @@ jobs:
           check-code: true
           force: ${{ matrix.experimental }}
 
+      - name: run api test
+        if: matrix.database == 'sqlite'
```

Comment:
Why does it require sqlite?

Reply:
I didn't get it to work with mysql and psql...
And I don't have a local test instance to figure out why, a ssh action for github actions also didn't work to debug the situation.

Reply:
Maybe the drivers are not loaded?

Reply:
Can you add a comment to that extend above/after the line? So we have an idea why later

Reply:
Yea I will do that, now that i have a working test in the other PR i think I will close this one, then test again if the test also works with the other DBs and then finalize the pr